lib/OnyxUtils.ts

```diff
@@ -72,6 +72,9 @@ let defaultKeyStates: Record<OnyxKey, OnyxValue<OnyxKey>> = {};
 let batchUpdatesPromise: Promise<void> | null = null;
 let batchUpdatesQueue: Array<() => void> = [];
 
+// Holds a map representing the latest data that was used to invoke a connection callback
```

Comment:
**non blocking thought:** maybe better to explain why we need this and what it is used for.